### ISSUE TITLE
Pri chybnej url sa resetujú niektoré polia v zozname

### ISSUE THREAD

**Issue by kocurvik** [TIS2023-FMFI/pracovne-cesty#136]
Dobrý deň,

do pola link na udalosť som dal pôvodne adresu cescg.org. Po pokuse o pridanie cesty som dostal oznam, že url nie je správna, tak som pred ňu pridal https://.

To fungovalo, ale resetovali sa polia: 
- Dopravný prostriedok
- Štát
- ŠPP Prvok
- Predpokladám, že aj účel cesty, ale ten som mal ako prvú možnosť

Omylom som tak poslal zlé údaje okrem ŠPP prvku, ktorý som si všimol. Výber polí by mal aj po chybe ostať tak ako som ho zvolil, alebo by som na to mal byť aspoň upozornený.

**Comment by Mcibula** [comment]
Dobrý deň, ďakujeme, pozrieme sa na to.

**Comment by pavelpetrovic** [comment]
Ano, selectom sa pri neuspesnej validacii resetuje vyber na prvy v zozname.

Tu niekde (a v ostatnych x-dropdown-input) asi treba povedat, ze ma pouzit "old value", ale neviem ako to funguje v Blade,
nasiel som navody iba pre obycajny select pre Laravel...

https://github.com/TIS2023-FMFI/pracovne-cesty/blob/b3fda0f3861d62def0944684f5d352495269966b/src/resources/views/business-trips/create.blade.php#L77C25-L77C43

**Comment by silviabielikova** [comment]
ta funkcionalita s old value je naprogramovana priamo v komponente dropdown-input (https://github.com/TIS2023-FMFI/pracovne-cesty/blob/b3fda0f3861d62def0944684f5d352495269966b/src/resources/views/components/dropdown-input.blade.php#L20), teda sa pouziva pre kazdy. dropdown-input je nas vlastny komponent, ktory som robila, aby ponukal uz vsetku funckonalitu a mal rovnake stylovanie. 
problem moze byt ale aj inde, napada mi napriklad validacia formularu (validuje sa po castiach). ale momentalne neviem. mozeme sa na to pozriek, ked bude viac casu, bohuzial teraz vobec nie je.

**Comment by pavelpetrovic** [comment]
@silviabielikova problem je v tom, ze ten ":selected" je pouzity iba v resources/views/business-trips/edit.blade.php
ale pocas zadavania cesty sa udaje nahadzuju cez resources/views/business-trips/create.blade.php
kde to pouzite nie je. ked som to tam len tak jednoducho skusal pridat, tak to padlo, lebo tam $trip nic nie je.
ani ked sa to schova za isset($trip) to nepomaha, tam to treba nejak inak. nemate nejaky hint kadial by mohla ist cesta?

**Comment by pavelpetrovic** [comment]
nejako som to teda v tomto jednom formulare navela navela v tu vysie odkazanom komite 
vyriesil, ale nebude to jediny formular, kde sa to deje.
este nieco podobne robi aj adminovi, skontrolujem neskor.
neviem preco, to neslo tak, ako to robite, blade je pre mna spanielska dedina...

**Comment by pavelpetrovic** [comment]
Described bugs were resolved.
